src/thor/triplegbuilder.cc

```diff
@@ -821,23 +821,33 @@ TripLeg_Edge* AddTripEdge(const AttributesController& controller,
 
   // Add names to edge if requested
   if (controller(kEdgeNames)) {
+#if 0
     std::vector<uint8_t> types;
-    auto names_and_types = edgeinfo.GetNamesAndTypes(types, true);
+#endif
+
+    auto names_and_types = edgeinfo.GetNamesAndTypes(true);
     trip_edge->mutable_name()->Reserve(names_and_types.size());
     std::unordered_map<uint8_t, std::pair<uint8_t, std::string>> pronunciations =
         edgeinfo.GetPronunciationsMap();
     uint8_t name_index = 0;
     for (const auto& name_and_type : names_and_types) {
+#if 0
       if (types.at(name_index) != 0) {
         // Skip the tagged names
         name_index++;
         continue;
       }
+#else
+      if (std::get<2>(name_and_type) != 0) {
+        // Skip the tagged names
+        continue;
+      }
+#endif
```

Comment:
im a little confused about these changes, im fine with the edge info name getting refactor to return a tuple instead of pass by reference. we can just delete the old way, or was there a good reason to keep it around?

Reply:
right, forgot to delete the old code,
(its in thor, and i unadvertedly added it to this PR)